docs/_config.yml

```diff
@@ -16,6 +19,16 @@ docs-api: http://docs.solace.com/API-Developer-Online-Ref-Documentation/swagger-
 docs-schema: http://docs.solace.com/SEMP/SEMP-API-Archit.htm#SEMP
 docs-semp-legacy: http://docs.solace.com/SEMP/Using-Legacy-SEMP.htm
 docs-vmr-setup: http://docs.solace.com/Solace-VMR-Set-Up/Setting-Up-VMRs.htm
+docs-router-concepts: http://docs.solace.com/Features/Core-Concepts.htm#router-concepts
+docs-mgmt-user-aa: http://docs.solace.com/Features/Mgmt-User-Authenticate-Authorize.htm
+docs-msg-vpn: http://docs.solace.com/Configuring-and-Managing-Routers/Configuring-VPNs.htm
+docs-client-profile: http://docs.solace.com/Features/Client-Authorization.htm#client_authentication_and_authorization_2866343001_219397
```

Comment:
This link will break easily. Should work with Solace docs team to get a permanent link to this location. This can be tracked as separate issue. Please raise as GitHub issue so we track it to a fix.